tests/unit/s2n_tls13_key_schedule_rfc8448_test.c

```diff
@@ -0,0 +1,381 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/* Needed to set up X25519 key shares */
+#include <openssl/evp.h>
+
+#include "s2n_test.h"
+#include "testlib/s2n_testlib.h"
+
+#include "tls/s2n_tls13_key_schedule.h"
+#include "tls/s2n_tls13_secrets.h"
+
+#include "tls/s2n_cipher_suites.h"
+
+struct s2n_cipher_suite *cipher_suite = &s2n_tls13_aes_128_gcm_sha256;
+
+const s2n_mode modes[] = { S2N_CLIENT, S2N_SERVER };
+
+int main(int argc, char **argv)
+{
+    BEGIN_TEST();
+
+    /* The RFC values use x25519,
+     * which is only supported via EVP APIs.
+     */
+    if (!s2n_is_evp_apis_supported()) {
+        END_TEST();
+    }
+
+    /*
+     * Simple 1-RTT Handshake
+     */
+    {
+        const uint32_t one_rtt_handshake_type = NEGOTIATED | FULL_HANDSHAKE;
+        const int one_rtt_message_nums[] = {
+                [SERVER_HELLO] = 1,
+                [SERVER_FINISHED] = 5,
+                [CLIENT_FINISHED] = 6,
+        };
+
+        /**
+         *= https://www.rfc-editor.org/rfc/rfc8448.html#section-3
```

Comment:
can you mark these as tests? I know they don't have implementations but it helps the categorize the references in the report.

```suggestion
         *= https://www.rfc-editor.org/rfc/rfc8448.html#section-3
         *= type=test
```

Reply:
Good catch, I completely forgot about the type=test option!